sections/06_discussion.md

```diff
@@ -183,32 +187,83 @@ multitudinous layers and architectures available
+[@tag:Kelley2016_basset]) and cleaned data (e.g., MoleculeNet
+[@tag:Wu2017_molecule_net]) publicly available to catalyze further research. As
+discussed in the Categorize section, there are complex privacy and legal issues
+involved in sharing patient data and deep learning models trained with it but
+also recent techniques to mitigate these concerns.  Furthermore, in some
+domains, some of the best training data has been generated privately, for
+example, high-throughput chemical screening data at pharmaceutical companies.
+There is little expectation or incentive for this private data to be shared.
+`TODO: this last line is fairly pessimistic, is there something better to say?
+Is it possible to create the right incentives and/or protect proprietary data?`
+
+Code sharing and open source licensing is essential for continued progress in
+this domain.  Even though some journals have improved requirements about code
+availability at the time of publication, there remain many opportunities to do
+better.  Preprints are firmly established in many areas of machine learning and
+are becoming increasingly popular in the biomedical domain.  We advocate that
+code repositories are made available alongside preprints instead of waiting
+until publication, especially because some important work in this field is never
+published in a conference or journal.  In addition, it is important for authors
+to share not only code for their core models but also scripts and code used for
+data cleaning (see above) and hyperparameter optimization.  These improve
+reproducibility and serve as documentation of the detailed decisions that impact
+model performance but may not be exhaustively captured in a manuscript's methods
+text. `TODO: there are many other things that could be added here (e.g.
+licensing), but I'm not sure much time to spend on issues that are not deep
+learning-specific; maybe cite some good existing work?)`
+
+Because many deep learning models are often built using one of several popular
+deep learning frameworks, it is also possible to directly share trained
+predictive models.  The availability of pre-trained models can accelerate
+research, with image classifiers as an apt example.  A pre-trained neural
+network can be quickly fine-tuned on new data and used in transfer learning,
+both discussed below.  `TODO: bring up image-encoding for variant classification
+that used pre-trained image classifier?` "Model zoos", collections of
+pre-trained models, are not yet common in biomedical domains but have started to
+appear in genomics applications [@tag:Angermueller2016_single_methyl
+@tag:Dragonn]. `TODO: other examples, possibly from Categorize or Treat?`
+
+DeepChem [@tag:AltaeTran2016_one_shot @tag:DeepChem @tag:Wu2017_molecule_net]
+and DragoNN [@tag:Dragonn] exemplify the benefits of sharing code under an open
+source license and pre-trained models. `TODO: it would be great to add more
```

Comment:
Source code:
https://github.com/greenelab/deep-review/issues/213 - Tagging @spoilt333 - don't think the raw data is here (looks like `/data/` path but not 100% sure). - did not see a license
https://github.com/greenelab/deep-review/issues/266 - tagging @zhangyan32 - looks like source but not models + data - did not see a license
https://github.com/greenelab/deep-review/issues/217 - source code, data can be downloaded - tagging @uci-cbcl - has license (GPL)
https://github.com/greenelab/deep-review/issues/171 tagging @CampagneLaboratory - this one has some docs, but not data/models if i'm seeing things correctly - has license (Apache)
https://github.com/greenelab/deep-review/issues/155 - code yes w/ docs. Don't think data. Tagging @traversc - just a quick note as I look through the source code I think some .pyc files made it in. not sure if that's intended. - no license
https://github.com/greenelab/deep-review/issues/150 - contains pre-trained models - tagging @raghavchalapathy - did not see a license
https://github.com/greenelab/deep-review/issues/32 - also has source code from @cui-cbcl + example file + data download + very nice readme - Custom license
https://github.com/greenelab/deep-review/issues/24 - source code + data [not version controlled]

That last one highlights to me that we don't _mention_ version control here but we probably should recommend that explicitly.